### PR TITLE
[wip] test of kubectl ref gen

### PR DESCRIPTION
**Pull request by kbhawkey**
- Testing simplified kubectl ref gen
- Updated static dependencies

### CODE REVIEW

**Comment by kbhawkey** [comment]
Looking for general impressions. Still working on styling fix ups, text formatting, other bits.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-18918--kubernetes-io-master-staging.netlify.com/docs/reference/generated/kubectl/kubectl-commands

**Comment by kbhawkey** [comment]
@julianvmodesto , I created this PR some time ago to revamp the kubectl command ref generation process and UI. The generation process is ready, though the styling and display is still a bit rough.
If you have time, I would appreciate your thoughts on this proposed update to the kubectl command reference.

**Comment by kbhawkey** [comment]
@tengqm

**Comment by kbhawkey** [comment]
@tengqm @sftim @zacharysarah , I would appreciate your comments on whether this draft of the kubectl command reference is a possible candidate for replacing the current reference. The information in the corresponding PR highlights some of the remaining todos (wish items).
The reference is generated using this code, https://github.com/kubernetes-sigs/reference-docs/pull/132

File: static/docs/reference/generated/kubectl/css/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    padding-top: 60px;
+}
+
+.body-content h1, .body-content h2 {
+    clear: both;
+    border-bottom: 3px solid lightslategrey;
+    padding-top: 20px;
+}
+
+/* reference title */
```
Comment (julianvmodesto) [inline static/docs/reference/generated/kubectl/css/stylesheet.css]: Nice, glad this is clear

File: static/docs/reference/generated/kubectl/css/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    background-color: rgb(51, 113, 227);
+    border-radius: 5px;
+}
+
+#sidebar-wrapper .strong-nav {
+    font-family: monospace;
+    font-weight: bold;
+}
+
+#sidebar-wrapper .nav-level-1.strong-nav {
```
Comment (julianvmodesto) [inline static/docs/reference/generated/kubectl/css/stylesheet.css]: It's hard to distinguish between the sidebar level 1 and level 2 headings. Could we make the separation more clear? Maybe size of level 1 or even add a separator?

**Comment by tengqm** [comment]
Comments posted to the reference-docs repo. Still some tweaks needed, but this is a good starting point.

**Comment by kbhawkey** [comment]
Closing again.
/close

**Pull request closed without merging**